multibase-conv/main.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	
+	multibase "github.com/multiformats/go-multibase"
+)
+
+func main() {
+	if len(os.Args) != 3 {
+		fmt.Printf("usage: %s CID NEW-BASE\n", os.Args[0])
```

Comment:
Do you think it would be better to make it `NEW-BASE CID...` this way it would work better with unix tooling.